src/site_upload/check_if_complete/check_if_complete.py

```diff
@@ -0,0 +1,140 @@
+import datetime
+import json
+import logging
+import os
+from time import sleep
+
+import awswrangler
+import boto3
+import botocore
+
+from shared import enums, functions
+
+logger = logging.getLogger()
+logger.setLevel("INFO")
+s3_client = boto3.client("s3")
+sns_client = boto3.client("sns")
+g_client = boto3.client("glue")
+
+
+def check_if_complete(message) -> (bool, dict):
+    transaction = functions.get_s3_json_as_dict(
+        bucket=os.environ.get("BUCKET_NAME"),
+        key=(
+            f"{enums.BucketPath.META.value}/transactions/{message['site']}__{message['study']}.json"
+        ),
+        s3_client=s3_client,
+    )
+    upload_time = datetime.datetime.fromisoformat(transaction["uploaded_at"])
+    for filetype, source, suffix in [
```

Comment:
Hmm, I guess same issue here around "forgetting to update a list of types" - maybe it's just something to live with. Or maybe there could be a central list of types? I dunno if it's worth optimizing around a forgetful future person or not. That person could also just search for `annotated_cube` and find all the spots.